integration_tests/integration/visitDetails.cy.ts

```diff
@@ -20,18 +20,31 @@ context('Visit details page', () => {
     const prisoner = TestData.prisoner()
     const { prisonerNumber: offenderNo } = prisoner
     const prisonerDisplayName = 'Smith, John'
-    const visit = TestData.visit({ createdTimestamp: '2022-02-14T10:00:00' })
+    const visit = TestData.visit({
```

Comment:
Making the test visit on this visit details page a `CANCELLED` one so that complete range of fields are now checked - e.g. cancellation reason banner at the top and all the bottom user details fields.